Cirumflex_Combined.ino

```diff
@@ -46,14 +53,15 @@ const int VAlRANGES[] = {0, 33,  58,  75,  86,  94, 100};
 //int n = 1;
 byte ZoneSet[4] = {1, 2, 3, 4}; // >>>change to: byte ZoneSet[4] = {0, 1, 2, 3};
 /////////////////////////////////////////////////////////////////////////
+
 void setup() {
   Serial.begin(9600);
   randomSeed(analogRead(RANDOMnoisePIN));
   for (byte led = 0; led < 3; led++) {
-    pinMode(aLunitPins[led], OUTPUT);
-    pinMode(bLunitPins[led], OUTPUT);
-    pinMode(cLunitPins[led], OUTPUT);
-    pinMode(dLunitPins[led], OUTPUT);
+    PWM.begin(aLunitPins[led]);
+    PWM.begin(bLunitPins[led]);
+    PWM.begin(cLunitPins[led]);
+    PWM.begin(dLunitPins[led]);
```

Comment:
my first guess at making "setup" work for native and TLC59711.

Reply:
Your guess is better than mine. Do we try it to find out?   
I will stop here for tonight. It took me ALL WEEK before I felt ready to dive into this (I am really slow)... I hope I am doing the right thing with these comments. 
I see a weird thing happened between lines 12 and 46. I'll check that out tomorrow (Sat Feb 8).

Reply:
Of course we should test! Hmmm. I didn't actually run the code.

Reply:
I do not have any "Brightness" or "Slowness" on the test box.